telegram/bot.py

```diff
@@ -1093,6 +1093,7 @@ def send_contact(self,
                      phone_number=None,
                      first_name=None,
                      last_name=None,
+                     vcard=None,
```

Comment:
Please move the argument to the last in the method signature (just before **kwargs).